validation/src/test/scala/hmda/validation/rules/lar/quality/Q005Spec.scala

```diff
@@ -0,0 +1,43 @@
+package hmda.validation.rules.lar.quality
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.{ BadValueUtils, LarEditCheckSpec }
+import org.scalacheck.Gen
+
+class Q005Spec extends LarEditCheckSpec with BadValueUtils {
+  property("All generated lars must pass") {
+    forAll(larGen) { lar =>
+      lar.mustPass
+    }
+  }
```

Comment:
I'm confused. Couldn't `larGen` in theory give you a LAR with a purchaserType of 3, a loan.propertyType of 2, and a loan.amount of 1200?